saas_test/tests/test_saas.py

```diff
@@ -77,11 +77,13 @@ def setUp(self):
         self.saas_operator_1 = self.env['saas.operator'].create({
             'type': 'local',
             'db_url_template': 'http://{db_name}.{db_id}.127.0.0.1.nip.io:8069',
+            'master_url': 'http://saas.127.0.0.1.nip.io:8069',
         })
 
         self.saas_operator_2 = self.env['saas.operator'].create({
             'type': 'local',
             'db_url_template': 'http://{db_name}.{db_id}.127.0.0.1.nip.io:8069',
+            'master_url': 'http://saas.127.0.0.1.nip.io:8069',
```

Comment:
"connect to build" doesn't not work on travis with such values

Reply:
Steps for tests:

* get authentication on master database in ``requests``' session
* open "connect to build ulr"
* check that you are finally redirected to ``/web`` page at the build

Reply:
For first step use ``authenticate`` method (try to don't copy-paste it) from ``odoo/tests/common.py``